KrishnaSharma206.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "@KrishnaSharma206"
+project: "An AI powered Robot"
+---
+
+# Object Recognition Warehouse Robot
+
+## Summary
+
+A robot that recohnizes certain shapes and navigates it's way accordingly, I am going to build this robot because I liked the idea of the amazon warehouse robot and the projects made by a guy named James Bruton, and I really wanna learn about object recognition through this project. I already have some experience making robots using arduino, so it will be something new for me.
+
+## Plan
+
+I will be using the jetson nano as the brain of this project, the raspberry pi wont be enough for this and the jetson nano will be a better alternative
+, I will be using a camera for the detection and I am planning to add more sensors and actuators to it in the future.
```

Comment:
Cool! What kind of sensors and actuators?

Reply:
I am planning to add a beanbag gripper to it so that it can grab items